libfreerdp/core/nla.c

```diff
@@ -1010,6 +1053,8 @@ static int nla_client_init(rdpNla* nla)
 		return -1;
 	}
 
+	nla_client_init_cred_handle(nla);
```

Comment:
Shouldn't we fail if it returns False ?

Reply:
I'll add a check, but I'll really need to make sure that whatever happens in it, it's considered a soft failure and will return true after throwing a warning.